### PR TITLE
Auto checkout

### PR DESCRIPTION
**Pull request by gforcada**
@tisto @mauritsvanrees is it ok to add the package that got a pull request merged on checkouts.cfg for **all** buildout.coredev branches which track the branch where the pull request was merged to?

Then we should clean them up whenever a release is done, right?

Fixes: https://github.com/plone/mr.roboto/issues/26

### CODE REVIEW

File: src/mr.roboto/src/mr/roboto/subscriber.py [hunk 1]
```diff
@@ -374,3 +379,101 @@ def warn_test_need_to_run(event):
                 version,
             )
         )
+
+
+def add_packge_to_checkouts(versions, package, event, short_url):
```
Comment (mauritsvanrees) [inline src/mr.roboto/src/mr/roboto/subscriber.py]: You may want to fix a typo in the function name: `packge` to `package`.

**Comment by mauritsvanrees** [comment]
Yes, seems good to me.

This code adds the new checkout at the bottom of `checkouts.cfg`. For 5.0 this would mean it ended up under the comment `# documentation only`. So two options:

1. Add a comment `# auto-added by mr.roboto` as bottom line. This is needed only once and we can do this manually of course, for all three current coredev branches. Every once in a while someone can go through the auto-added list and move the package names to their proper place.
2. Remove all comments and just have one list. Documentation-only, test-only and translation-only fixes are important too, and those comments may be wrong once a second pull request for a package gets merged.

Both are fine with me. I would choose option 2. But we could start with option 1 and see how it goes. Neither option requires a code change.

With option 2 it may be feasible to sort the checkouts automatically, though this can be tricky when the `checkouts.cfg` is subtly different than what we expect. Can be done later.

**Comment by gforcada** [comment]
@mauritsvanrees thanks for the feedback!

I forgot to bring that topic (appending at the bottom of the file), but you already discussed it.

I'm also more in favor of 2, but having a short period with 1 would be useful as well so we see how many packages were auto-added.

This pull request still misses to change the automatic email that is sent and that was asking for action to the one that pushed code, although if it's not a pull request it will not be added.

On that topic (commits directly to master/tracked branch) we should try to also auto-commit to buildout.coredev on this situation, although, let's battle test this pull request first and later we can think of further improvements.

Sounds good?

File: src/mr.roboto/src/mr/roboto/subscriber.py (hunk 1 above)
Comment (gforcada) [inline src/mr.roboto/src/mr/roboto/subscriber.py]: Fixed!

**Comment by mauritsvanrees** [comment]
Yes, sounds good. Thanks for working on this!

**Pull request merged**